package-lock.json

```diff
@@ -5403,7 +5403,7 @@
         "wrappy": {
             "version": "1.0.2",
             "resolved": "https://registry.npmjs.org/wrappy/-/wrappy-1.0.2.tgz",
-            "integrity": "sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8="
+            "integrity": "sha512-l4Sp/DRseor9wL6EvV2+TuQn63dMkPjZ/sp9XkghTEbV9KlPS1xUsZ3u7/IQO4wxtcFB4bgpQPRcR3QCvezPcQ=="
```

Comment:
I don't understand why package-lock needed updating in this PR - we didn't touch packages. Seems benign, but please doublecheck.

Reply:
`package-lock.json` simply updated after running `npm i` due to package updates. dependencies havent been updated since commit 9 months ago: [here](https://github.com/Exabyte-io/esse/commit/89b9301354d6a03c51ef9e4453da4b236208a61e)